model-archiver/README.md

```diff
@@ -138,12 +138,14 @@ A serialized file (.pt or .pth) should be a checkpoint in case of torchscript an
 
 ### Handler
 
-Handler can be TorchServe's inbuilt handler name or path to a py file to handle custom TorchServe inference logic. TorchServe supports following handlers out or box:
+Handler can be TorchServe's inbuilt handler name or path to a py file to handle custom TorchServe inference logic. Some examples of supported handlers
```

Comment:
I think this might be confusing. This below are indeed the 4 default handlers that we support and come bundled with the TS python module. The examples are different than these handlers.